frontend/src/Judge/queries/submissions.graphql

```diff
@@ -1,8 +1,20 @@
 query Submissions($id: ID!) {
   submissions: entries(showId: $id) {
-    id,
+    id
+    title
+    entryType
     show {
       id
     }
+    ... on Video {
```

Comment:
I like this syntax ... could you possibly also convert `frontend/src/Student/queries/shows.graphql` to use it as well? If you don't have time that's fine, don't let it block the merge.